crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,6 +1156,19 @@ impl World {
         result
     }
 
+    /// "Sends" an `event`.
+    #[inline]
+    pub fn event_send<E: crate::event::Event>(&mut self, event: E) {
+        self.resource_mut::<crate::event::Events<E>>().send(event);
```

Comment:
I think this should use the get_* version and log a warn to remember to call add_event

Reply:
![image](https://user-images.githubusercontent.com/2632925/179421129-689e7b05-1170-4769-96f0-ea314fb90f75.png)